bedrock/mozorg/models.py

```diff
@@ -55,3 +61,22 @@ class WebvisionDoc(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class AboutPage(AbstractStructuralPage):
+    max_count = 1  # Ensure there's only one instance of this page
+    subpage_types = ["LeadershipPage"]
+
+
+class LeadershipPage(AbstractBedrockCMSPage):
+    max_count = 1  # Ensure there's only one instance of this page
+    # parent_page_types = ["AboutPage"] - WHY DOES THIS NOT WORK?
```

Comment:
**issue:** I'm not sure why this doesn't work? Might it be an artifact from `AbstractBedrockCMSPage`?